applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -300,6 +305,14 @@ def __main():
         default=False,
         help="Set when using nRF21540 for extra TX power",
         )
+    parser.add_argument(
+        "-u"
+        "--user_bt_name",
+        action="store_true",
+        dest="user_specific_name",
```

Comment:
```suggestion
        dest="user_bt_name",
```
Unless there's a specific reason I think it's better to have the flag and the variable have the same naming